src/python/provisioner/src/BUILD

```diff
@@ -0,0 +1,8 @@
+python_library()
+
+python_awslambda(
+    name="lambda",
```

Comment:
a) is this really the right name for it?

b) thank you for writing a prod provisioner service. can I assume it's on your plate to remove grapl_provision (or rename it, or whatever its fate is)?
(a lot of the code is duplicated in there like meta_into_edge etc etc)

Reply:
(a) I'm not sure exactly what `name` does here, I stuck *very* closely to the `pants` docs and it all worked out so I'm a little reluctant to change anything.

(b) I have no idea what `grapl_provision` is, or what it does. I understand @christophermaier  is working on related changes in Pulumi right now as well. It would be a good idea to synchronize further developments of the provisioning code. I expect eventually everything will end up being Pulumi under the hood with a `graplctl` facade, but I'm not 100% clear on the details of how we'll get there.